wps-common/src/main/java/au/org/aodn/aws/wps/request/ExecuteRequestHelper.java

```diff
@@ -34,6 +38,36 @@ public boolean hasRequestedOutput(String identifier) {
         return false;
     }
 
+    public void validateInputs() throws ValidationException {
+        String layerName = getLiteralInputValue("layer");
+        String subset = getLiteralInputValue("subset");
+        String email = getEmail();
+
+        if (layerName == null) {
+            throw new ValidationException("Request must have a layer name");
+        }
+
+        if (subset == null) {
+            throw new ValidationException("Request must have a subset");
+        } else {
+            int latLonCount = 0;
+            Pattern latLonPattern = Pattern.compile("([+-]?\\d+\\.?\\d+)\\s*,\\s*([+-]?\\d+\\.?\\d+)");
```

Comment:
Should probably align better with aggregation validation https://github.com/aodn/aws-wps/blob/master/aggregation-worker/src/main/java/au/org/emii/geoserver/client/SubsetParameters.java#L40 - we haven't used regexes for some time - they were problematic

Reply:
Even better if they used the same code so they couldn't get out of synch.

Reply:
Yeah that's a really good point. I'll amend that